src/ui/control/attribution_control.js

```diff
@@ -53,7 +54,12 @@ class AttributionControl {
 
         this._map = map;
         this._container = DOM.create('div', 'mapboxgl-ctrl mapboxgl-ctrl-attrib');
+        this._compactButton = DOM.create('button', 'mapboxgl-ctrl-attrib-button', this._container);
+        this._compactButton.addEventListener('click', this._toggleAttribution);
+        this._setButtonTitle(this._compactButton, 'ToggleAttribution');
+        this._compactButton.setAttribute('role', 'button');
```

Comment:
This seems redundant as `<button>` elements have an [implicit `role` of `button`](https://www.w3.org/TR/html-aria/#el-button).